sdk/dotnet/Ec2/Outputs/GetLaunchTemplateNetworkInterfaceResult.cs

```diff
@@ -13,7 +13,7 @@ namespace Pulumi.Aws.Ec2.Outputs
     [OutputType]
     public sealed class GetLaunchTemplateNetworkInterfaceResult
     {
-        public readonly bool AssociatePublicIpAddress;
+        public readonly string AssociatePublicIpAddress;
```

Comment:
That's a breaking change, I guess an intentional change on the TF side?

Reply:
This is now fixed :)